client/src/components/JurisdictionAdmin/BatchRoundDataEntry.tsx

```diff
@@ -0,0 +1,645 @@
+              selectedTabId={selectedTabId}
+            >
+              <Tab
+                id={VOTE_TOTALS_TAB_ID}
+                key={VOTE_TOTALS_TAB_ID}
+                panel={
+                  <BatchResultTallySheet
+                    areResultsFinalized={areResultsFinalized}
+                    cancelSheetRename={cancelSheetRename}
+                    confirmSheetRename={confirmSheetRename}
+                    contest={contest}
+                    deleteSheet={() => deleteSheet(0)}
+                    enableEditing={enableEditing}
+                    formMethods={formMethods}
+                    index={0}
+                    initiateSheetRename={() => initiateSheetRename(0)}
+                    isEditing={isEditing}
+                    isRenamingSheet={isRenamingSheet}
+                    isTotalsSheet={sheets.length > 1}
+                    savedResults={
+                      selectedBatch.resultTallySheets[0]?.results || {}
+                    }
+                    saveResults={saveResults}
+                    sheets={sheets}
+                  />
+                }
+              >
+                Vote Totals
+              </Tab>
+              {sheets.length > 1 &&
+                sheets.map((sheet, i) => (
+                  <Tab
+                    id={sheet.name}
+                    key={sheet.id}
+                    panel={
+                      <BatchResultTallySheet
+                        areResultsFinalized={areResultsFinalized}
+                        cancelSheetRename={cancelSheetRename}
+                        confirmSheetRename={confirmSheetRename}
+                        contest={contest}
+                        deleteSheet={() => deleteSheet(i)}
+                        enableEditing={enableEditing}
+                        formMethods={formMethods}
+                        index={i}
+                        initiateSheetRename={() => initiateSheetRename(i)}
+                        isEditing={isEditing}
+                        isRenamingSheet={isRenamingSheet}
+                        savedResults={
+                          selectedBatch.resultTallySheets[i]?.results || {}
+                        }
+                        saveResults={saveResults}
+                        sheets={sheets}
+                      />
+                    }
+                  >
+                    {sheetNameBeforeRename && sheetNameBeforeRename.index === i
+                      ? sheetNameBeforeRename.name
+                      : watch(`resultTallySheets[${i}].name`)}
+                  </Tab>
+                ))}
+
+              <Tabs.Expander />
+
+              {isEditing && (
+                <Button
+                  disabled={isRenamingSheet}
+                  icon="add"
+                  minimal
+                  onClick={addSheet}
+                >
+                  {sheets.length > 1
+                    ? 'Add sheet'
+                    : 'Use multiple tally sheets'}
```

Comment:
Even though I like the way these sentence case button labels look, I think maybe we should title case them for consistency with the rest of the UI. If I could go back, I would make them all sentence case...